bench/wtperf/wtperf_opt.i

```diff
@@ -153,7 +153,7 @@ DEF_OPT_AS_BOOL(
   reopen_connection, 1, "close and reopen the connection between populate and workload phases")
 DEF_OPT_AS_UINT32(
   report_interval, 2, "output throughput information every interval seconds, 0 to disable")
-DEF_OPT_AS_UINT32(run_ops, 0, "total read, insert and update workload operations")
+DEF_OPT_AS_UINT32(run_ops, 0, "total insert, modify, insert and update workload operations")
```

Comment:
"insert, modify, insert" - one of the 'insert' should be 'read'

Reply:
Fixed.